cypress/integration/apps/citas/turnos/punto_inicio.spec.js

```diff
@@ -1,78 +1,161 @@
-context('Agenda dinamicas', () => {
-    let token
+context('punto de inicio', () => {
+    let token;
     before(() => {
         cy.login('30643636', 'asd').then(t => {
             token = t;
-            cy.createAgenda('apps/citas/turnos/agendaDinamicaDarTurno', 0, 0, 1, token);
-            cy.createAgenda('apps/citas/turnos/agendaDarTurnoProgramado', 8, null, null, token);
+            
         });
     });
-
-
-    it('dar turno agenda dinámica', () => {
+    beforeEach(() => {
         cy.server();
-        cy.route('GET', '**api/core/mpi/pacientes?**').as('busquedaPaciente');
-        cy.route('PATCH', '**/api/modules/turnos/turno/**').as('darTurno');
-        cy.route('GET', '**/api/modules/turnos/agenda?rango=true&desde=**').as('cargaAgendas');
-        cy.route('GET', '**api/core/tm/tiposPrestaciones**').as('prestaciones');
-
         cy.goto('/citas/punto-inicio', token);
+        cy.route('GET', '**api/core/mpi/pacientes?**').as('busquedaPaciente');
+        cy.route('GET', '**api/core/log/paciente?idPaciente=**').as('seleccionPaciente');
+        
+    })
+    
+    it('Buscar paciente inexistente', () => {
+        cy.get('plex-text input[type=text]').first().type('12362920').should('have.value', '12362920');
+        cy.wait('@busquedaPaciente').then((xhr) => {
+            expect(xhr.status).to.be.eq(200);
+        });
+        cy.get('.alert.alert-danger').should('contain',  'No se encontró ningún paciente..');
+        
+    });
+    
+    it('dar turno', () => { 
+        // Borro los datos de la base antes de los test
+        cy.exec('npm run prod:reset');
```

Comment:
```suggestion
        cy.seed()
```

¿Porque lo hacen en particular en este test?

Reply:
ok